src/views/landing-page.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Route, Switch } from "react-router-dom";
+
+import Login from "../containers/login";
+import Homepage from "./homepage";
+
+const LandingPage = () => {
+  return (
+    <div>
+      <Switch>
+        <Route path="/" exact component={Homepage} />
+        <Route path="/login" exact component={Login} />
+      </Switch>
+    </div>
+  );
+};
+
+export default LandingPage;
```

Comment:
@Nicanor008 this is okay, although according to file naming conventions, the file name should be the same as the components name and in PascalCase. In this case, the file name should be `LandingPage.jsx`, not `landing-page.js`.

Refer to: https://github.com/airbnb/javascript/tree/master/react#naming

Reply:
got it, let me make those changes @victorjambo